spec/models/conference_spec.rb

```diff
@@ -58,6 +58,28 @@
 
       expect(DateFormatter).to have_received(:new).with(start_date, end_date)
       expect(formatter).to have_received(:to_s)
+      end
+  end
+
+  describe "geocoded_by" do
+    it "is geocoded_by location" do
+      expected_latitude = 40.7143528
+      expected_longitude = -74.0059731
+
+      conference = create(:conference)
+      Geocoder::Lookup::Test.add_stub(
+        conference.location, [
+          {
+            'latitude'     => expected_latitude,
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.